### PR TITLE
fixing broken link in Setting up a payment method blog

### PR DESCRIPTION
**Pull request by digi-booster**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

**Comment by samthor** [comment]
I've closed some of your PRs because they don't fix issues.

However, for the others, you need to sign our CLA before we can accept your changes, see the bot's comment.

**Comment by digi-booster** [comment]
> I've closed some of your PRs because they don't fix issues.
> 
> However, for the others, you need to sign our CLA before we can accept your changes, see the bot's comment.

Can I report the issue of 90+ broken links?

**Comment by digi-booster** [comment]
@googlebot I signed it!

**Comment by samthor** [comment]
Feel free to fix broken links :)

File: src/site/content/en/payments/setting-up-a-payment-method/index.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ payment app appears in the Payment Request UI.
         </td>
         <td>
         Used to detect the app that acts as the OS-provided payment app.
-        Find more details at <a href="/native-payment-apps-overview">Android
+        Find more details at <a href="https://web.dev/android-payment-apps-developers-guide/">Android
```
Comment (samthor) [inline src/site/content/en/payments/setting-up-a-payment-method/index.md:325]: Can you remove "https://web.dev" from this, and just start with "/"?

**Pull request merged**